third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -784,6 +787,55 @@ func resourceContainerCluster() *schema.Resource {
 	}
 }
 
+// Setting 0 for guest accelerator count has the effect of removing all of the guest_accelerators. Since the API
```

Comment:
```suggestion
// Setting 0 for guest accelerator type's count has the effect of removing all of the guest_accelerators of that type. Since the API
```